developer_manual/design/popovermenu.rst

```diff
@@ -0,0 +1,68 @@
+.. sectionauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+.. codeauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+..  _popovermenu:
+
+===============
+Popover Menu
+===============
+
+What is a popover menu
+=======================
+
+This is a quick menu that open on click. We're usually using the three dots icons on nextcloud.
+
+This is exactly the same as the :ref:`navigation menu <navigation_menu>`. The only difference is the popovermenu class.
+
+.. image:: ../images/popovermenu.png
+
+Basic layout
+=============
+
+.. code-block:: html
+
+	<div class="popovermenu">
+	    <ul>
+	        <li>
+	            <a href="#" class="icon-details">
+	                <span>Details</span>
+	            </a>
+	        </li>
+	        <li>
+	            <button class="icon-details">
+	                <span>Details</span>
+	            </button>
+	        </li>
+	        <li>
+	            <button>
+	                <span class="icon-details"></span><span>Details</span>
+	            </button>
+	        </li>
+	        <li>
+	            <a>
+	                <span class="icon-details"></span><span>Details</span>
+	            </a>
+	        </li>
+	    <ul>
+	</div>
+
+Technical details
+==================
+
+* The only allowed menu items elements are **a** and **button**.
+* You can mix between a and button on the same menu (in case of form or direct link) like the example above
+* You need to put the entire menu just after the three dot icon ``<div><span class="icon-more"></span><div class="popovermenu">...</div></div>``
+* You do not need js, css only is ok for positioning. Js is **still** required to handle the hide/show.
+* Only **one** ul is allowed.
+* Only **one level** of menu is allowed.
+* Every entry **needs** to have its own icon. This greatly improves the UX.
+* The required **right** distance to the border (or padding, whatever you want to use) of the three-dot icon should be 14px (5 for menu margin and 6 for arrow position)
```

Comment:
We should do that automatically though, right? We shouldn’t need to manually mention any px values here?

Reply:
It is done automatically. This information is for the menu button. Not everyone will use a standard button. For example again, the federationscope menu, the button is not 44px width. :)